test/tools/wee8_compile/wee8_compile.cc

```diff
@@ -0,0 +1,241 @@
+// NOLINT(namespace-envoy)
```

Comment:
Note: This is a standalone tool, that should be moved to V8 at some point (there is nothing Envoy or Proxy-WASM specific about it), so it doesn't depend on any Envoy codebase.

Reply:
Can you add some high level block comments about this tool. What does it do? Why does it exist? Don't other WASM users have a similar problem to whatever this tool is doing?

Reply:
Added. PTAL.

Reply:
This should be under `tools` instead of `test/tools` then?

Reply:
`tools/` seems to be used exclusively for build tools.

`test/tools`, on the other hand, contains a few "real" helper tools, just like this one.

Reply:
Let me know how you want to resolve that (alternative, if we want to move this tool to a separate repo, then this is moot).